### PR TITLE
MONIT-32274: Update Beachshirts log output to acceptable sample logs.

### PR DESCRIPTION
**Pull request by xuranchen**

### CODE REVIEW

File: shopping/src/main/java/com/wfsample/shopping/ShoppingService.java [hunk 1]
```diff
@@ -40,6 +42,7 @@
  * @author Srujan Narkedamalli (snarkedamall@wavefront.com).
  */
 public class ShoppingService extends Application<DropwizardServiceConfig> {
+  static Logger logger =  LogManager.getLogger(ShoppingService.class);
```
Comment (deosu) [inline shopping/src/main/java/com/wfsample/shopping/ShoppingService.java:45]: Just trying to understand- In the ShoppingService class why we are logging with `logger.warn()`, whereas in other classes we log using `WavefrontTracer`?

**Review by deosu — APPROVED** [review]
LGTM. Just a small comment for clarification.

**Comment by deosu** [comment]
@xuranchen, also it would be great if you can give an overview of how these logs will be displayed on wavefront UI. We can use the pairing session to go over this. The changes in the PR look good to me.

**Comment by amitw-vmware** [comment]
lgtm

**Review by amitw-vmware — APPROVED** [review]
lgtm

**Review by amitw-vmware — APPROVED** [review]
lgtm2

**Pull request merged**